developer-docs-site/docs/reference/faq.md

```diff
@@ -0,0 +1,328 @@
+
+### *What changed from Diem to Aptos?*
+
+We have focused upon:
+
+  * going from permissioned to permissionless → proof of authority to proof of stake
+  * encouraging safety and performance
+  * making major changes to storage
+  * clear path to 100k TPS 
+
+## Aptos Community
+
+### How can I find you on Telegram, Discord, Twitter and other communities?
+
+- Telegram
+    - Aptos Labs: [https://t.me/AptosLabs](https://t.me/AptosLabs)
+    - Aptos Official (announcements): [https://t.me/Aptos_official](https://t.me/Aptos_official)
+- Discord: [https://discord.gg/tuDs9TmWwv](https://discord.gg/tuDs9TmWwv)
+- Twitter: [https://twitter.com/aptoslabs](https://twitter.com/aptoslabs)
+- Twitch: [https://www.twitch.tv/aptoslabs](https://www.twitch.tv/aptoslabs)
+
+### How do I keep up with news from Aptos Labs?
+
+You can see all Aptos news in the Aptos Discord channels [#announcements](https://discord.com/channels/945856774056083548/946124073400938497) or in [#social-media](https://discord.com/channels/945856774056083548/950849051358822431).
+
+## Aptos Discord Community Roles
+
+### What are roles?
+
+A role gives a specific level of permission. As more and more contributors join Aptos Discord community, we created a few roles so that the community is better organized. 
+
+Check out the [Aptos Discord channel](https://discord.com/channels/945856774056083548/957486688710561872) `#roles` to get the role for yourself. 
+
+### What do these roles mean?
+
+- Node Operator: Any Aptos community member running Aptos nodes.
+- Developer: Any Aptos community member developing on Aptos Blockchain.
+- Aptonaut: Anyone who is exploring opportunities to contribute in the [Aptos Discord channel](https://discord.com/channels/945856774056083548/963777510473084988).
+
+### How do I get roles?
+
+Head over to the [Aptos Discord channel](https://discord.com/channels/945856774056083548/957486688710561872) `#roles` to get the role for yourself by reacting with the designated emoji. 
+
+## Aptos Token
+
+### Does Aptos have a token?
+
+There is no Aptos token. 
+
+### Is there an Aptos token for sale, presale or airdrop?
+
+There is no current token sale. Anyone claiming otherwise is running a scam and please make sure to report them in the [Aptos Discord channel](https://discord.com/channels/945856774056083548/951160191716585542) `#report-scams`. 
+
+Aptos Labs will post any updates and announcements in the [Aptos Discord channel](https://discord.com/channels/945856774056083548/946124073400938497) `#announcements`.
+
+## Working with Aptos Labs
+
+### I want to be a translator, what are my choices?
+
+Currently, any translation of Aptos Labs documentation from English to a non-English language is a community effort. Check the [Aptos Discord](https://discord.gg/zTDYBEud7U) channel.
```

Comment:
do we actually have anything in this space? it feels like this isn't an adequate response. A more reasonable response seems like "The team is currently determining the best route to integrate internationalization into aptos.dev" or something